features/support/env.js

```diff
@@ -108,7 +97,7 @@ module.exports = function () {
         var verify = (binPath, cb) => {
             fs.exists(binPath, (exists) => {
                 if (!exists) return cb(new Error(util.format('%s is missing. Build failed?', binPath)));
-                var helpPath = util.format('%s --help > /dev/null 2>&1', binPath);
+                var helpPath = util.format('%s --help', binPath);
```

Comment:
`/dev/null` is an incorrect device, but stdout is simply ignored in `child_process.exec` callback